MountainArr_java_Pratheeksha

```diff
@@ -0,0 +1,35 @@
+import java.io.*;
+import java.lang.*;
+import java.util.*;
+
+class GFG {
```

Comment:
Looks like a Code from GeeksForGeeks ;)
It would be great if you write the code yourself, and explaining why the functionalities are required using comments.